cob_cartesian_controller/scripts/action_test.py

```diff
@@ -0,0 +1,425 @@
+    goal.move_type = CartesianControllerGoal.LIN
+    goal.move_lin.x = 1.0
+    goal.move_lin.y = 0.0
+    goal.move_lin.z = 0
+    goal.move_lin.roll = 0.0
+    goal.move_lin.pitch = 0.0
+    goal.move_lin.yaw = 0.0
+    goal.move_lin.profile.vel = 0.1
+    goal.move_lin.profile.accl = 0.2
+    goal.move_lin.profile.profile_type = Profile.SINOID
+    goal.move_lin.rotate_only = False
+    client.send_goal(goal)
+    client.wait_for_result()
+
+    rospy.sleep(2.0)
+
+    # Step 5
+    goal = CartesianControllerGoal()
+    goal.move_type = CartesianControllerGoal.LIN
+    goal.move_lin.x = 0.0
+    goal.move_lin.y = -1.0
+    goal.move_lin.z = 0
+    goal.move_lin.roll = 0.0
+    goal.move_lin.pitch = 0.0
+    goal.move_lin.yaw = 0.0
+    goal.move_lin.profile.vel = 0.1
+    goal.move_lin.profile.accl = 0.2
+    goal.move_lin.profile.profile_type = Profile.SINOID
+    goal.move_lin.rotate_only = False
+    client.send_goal(goal)
+    client.wait_for_result()
+
+    rospy.sleep(2.0)
+
+    # Step 6
+    goal = CartesianControllerGoal()
+    goal.move_type = CartesianControllerGoal.LIN
+    goal.move_lin.x = -1.0
+    goal.move_lin.y = 1.0
+    goal.move_lin.z = 0
+    goal.move_lin.roll = 0.0
+    goal.move_lin.pitch = 0.0
+    goal.move_lin.yaw = 0.0
+    goal.move_lin.profile.vel = 0.1
+    goal.move_lin.profile.accl = 0.2
+    goal.move_lin.profile.profile_type = Profile.SINOID
+    goal.move_lin.rotate_only = False
+    client.send_goal(goal)
+    client.wait_for_result()
+
+    rospy.sleep(2.0)
+
+    # Step 7
+    goal = CartesianControllerGoal()
+    goal.move_type = CartesianControllerGoal.LIN
+    goal.move_lin.x = 0.0
+    goal.move_lin.y = -1.0
+    goal.move_lin.z = 0
+    goal.move_lin.roll = 0.0
+    goal.move_lin.pitch = 0.0
+    goal.move_lin.yaw = 0.0
+    goal.move_lin.profile.vel = 0.1
+    goal.move_lin.profile.accl = 0.2
+    goal.move_lin.profile.profile_type = Profile.SINOID
+    goal.move_lin.rotate_only = False
+    client.send_goal(goal)
+    client.wait_for_result()
+
+    rospy.sleep(2.0)
+
+    # Step 8
+    goal = CartesianControllerGoal()
+    goal.move_type = CartesianControllerGoal.LIN
+    goal.move_lin.x = 1.0
+    goal.move_lin.y = 1.0
+    goal.move_lin.z = 0
+    goal.move_lin.roll = 0.0
+    goal.move_lin.pitch = 0.0
+    goal.move_lin.yaw = 0.0
+    goal.move_lin.profile.vel = 0.1
+    goal.move_lin.profile.accl = 0.2
+    goal.move_lin.profile.profile_type = Profile.SINOID
+    goal.move_lin.rotate_only = False
+    client.send_goal(goal)
+    client.wait_for_result()
+
+#    goal.move_type = CartesianControllerGoal.LIN
+#    goal.move_lin.x = -0.9
+#    goal.move_lin.y = 0
+#    goal.move_lin.z = 0
+#    goal.move_lin.roll = 0
+#    goal.move_lin.pitch = 0
+#    goal.move_lin.yaw = 0
+#    goal.move_lin.rotate_only = False
+#
+#    goal.move_lin.profile.profile.vel = 0.2
+#    goal.move_lin.profile.profile.accl = 0.1
+#    goal.move_lin.profile.profile_type = Profile.SINOID
+
+if __name__ == '__main__':
```

Comment:
Do we still use this script? Or is it superseded by those in `cob_twist_controller/scripts/test`?
If it's not needed, I'd remove it...or at least split it maybe...